packages/superset-ui-plugin-chart-word-cloud/src/chart/WordCloud.tsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import cloudLayout, { Word } from 'd3-cloud';
+import { PlainObject } from 'encodable/lib/types/Data';
+import { WordCloudEncoding, wordCloudEncoderFactory } from './Encoder';
+
+export const ROTATION = {
+  flat: () => 0,
+  /* eslint-disable-next-line no-magic-numbers */
+  random: () => Math.floor(Math.random() * 6 - 3) * 30,
```

Comment:
Maybe leave a comment to explaining that this calculates a random rotation between -90 and 90 degrees. I'm assuming the unit is degrees here since square is `* 90`, but it would be helpful when skimming the code.